### PR TITLE
Wiser and more informative logger; daemonize support 

### PR DESCRIPTION
**Pull request by ZenithalHourlyRate**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,34 +21,48 @@ NAME:
 USAGE:
    auth-thu [options]
    auth-thu [options] auth [auth_options]
+   auth-thu [options] deauth [auth_options]
    auth-thu [options] login
    auth-thu [options] logout
+   auth-thu [options] onlien [online_options]
```
Comment (z4yx) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -21,34 +21,48 @@ NAME:
...
        OPTIONS:
-           --ip value      authenticating for specified IP address
-           --no-check, -n  skip online checking, always send login request
-           --logout, -o    log out of the online account
-           --ipv6, -6      authenticating for IPv6 (auth6.tsinghua)
-           --host value    use customized hostname of srun4000
-           --insecure      use http instead of https
+         --ip value         authenticating for specified IP address
+         --no-check, -n     skip online checking, always send login request
+         --logout, -o       de-auth of the online account (for compatibility)
```
Comment (z4yx) [inline README.md]: "behaves the same as deauth command, for backward-compatibility" would be more clear?

File: cli/main.go [hunk 3]
```diff
@@ -365,14 +405,27 @@ func main() {
 				Flags: []cli.Flag{
 					&cli.StringFlag{Name: "ip", Usage: "authenticating for specified IP address"},
 					&cli.BoolFlag{Name: "no-check, n", Usage: "skip online checking, always send login request"},
-					&cli.BoolFlag{Name: "logout, o", Usage: "log out of the online account"},
+					&cli.BoolFlag{Name: "logout, o", Usage: "de-auth of the online account (for compatibility)"},
```
Comment (z4yx) [inline cli/main.go]: ↑

**Review by z4yx — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ZenithalHourlyRate) [reply README.md]: applied

**Review by z4yx — APPROVED** [review]

**Pull request merged**